### PR TITLE
Schedule implemented, speakers added.

### PR DESCRIPTION
**Pull request by wblankenship**

### CODE REVIEW

File: views/schedule.handlebars [hunk 1]
```diff
@@ -1,40 +1,46 @@
...
-        <li>21:00 - 22:00 - Judging</li>
+      <li>Doors open at 5:00 pm</li>
+      <li>06:15 pm - 06:25 pm - William Blankenship - Opening</li>
+      <li>06:25 pm - 06:45 pm - Jeremy Packer - Innovation in Southern Illinois</li>
+      <!--<li>06:25 pm - 06:45 pm - <a href="/schedule/packer">Jeremy Packer - Innovation in Southern Illinois</a></li>-->
+      <li>06:45 pm - 07:05 pm - TBA</li>
+      <li>07:05 pm - 07:25 pm - <a href="/schedule/schneider">Zach Schneider - Remote Teams: Brining opportunities to Southern Illinois</a></li>
+      <li>07:30 pm - 08:00 pm - Dinner (Thanks <a href="http://www.hacksi.me/schedule">Little Caesars</a> and <a href="https://twitter.com/davglass">@davglass</a>!)</li>
+      <li>08:00 pm - 12:00 am - HACKING</li>
+      <li>21:00 - 22:00 - Judging</li>
```
Comment (davglass) [inline views/schedule.handlebars]: This shouldn't say Judging here right?

File: views/schedule.handlebars [hunk 2]
```diff
@@ -1,40 +1,46 @@
...
+      <li>12:30 pm - 06:00 pm - EVEN MORE HACKING</li>
+      <li>06:30 pm - 07:00 pm - Dinner (Thanks <a href="http://www.hacksi.me/schedule">Little Caesars</a> and <a href="https://twitter.com/davglass">@davglass</a>!)</li>
+      <li>07:00 pm - 07:30 pm - MORE HACKING</li>
+    </ul>
+    <h3>Sunday, February 1st 2015</h3>
+    <ul>
+      <li>12:00 am - 07:00 am - MORE HACKING</li>
+      <li>07:00 am - 07:30 am - Breakfast (Thanks <a href="http://www.greplytix.com/">greplytix</a>!)</li>
+      <li>07:30 am - 11:30 am - MORE HACKING</li>
+      <li>11:30 am - 12:00 pm - Lunch (Thanks <a href="http://www.hacksi.me/schedule">Little Caesars</a> and <a href="https://twitter.com/davglass">@davglass</a>!)</li>
```
Comment (davglass) [inline views/schedule.handlebars]: LC url is wrong.

File: views/schedule.handlebars [hunk 3]
```diff
@@ -1,40 +1,46 @@
...
+    <h3>Saturday, January 31st 2015</h3>
     <ul>
-        <li>Arduino Hacking</li>
-        <li>NodeCopter Flights/Lessons</li>
+      <li>12:00 am - 07:00 am - MORE HACKING</li>
+      <li>07:00 am - 07:30 am - Breakfast (Thanks <a href="http://www.greplytix.com/">greplytix</a>!)</li>
+      <li>07:30 am - 12:00 pm - MORE HACKING</li>
+      <li>12:00 pm - 12:30 pm - Lunch (Thanks <a href="http://www.greplytix.com/">greplytix</a>!)</li>
+      <li>12:30 pm - 06:00 pm - EVEN MORE HACKING</li>
+      <li>06:30 pm - 07:00 pm - Dinner (Thanks <a href="http://www.hacksi.me/schedule">Little Caesars</a> and <a href="https://twitter.com/davglass">@davglass</a>!)</li>
```
Comment (davglass) [inline views/schedule.handlebars]: LC url is wrong.

File: views/schedule.handlebars (hunk 1 above)
Comment (wblankenship) [inline views/schedule.handlebars]: Nope, remnants from last year. Fix imminent.

**Pull request merged**